package.json

```diff
@@ -2,7 +2,54 @@
   "name": "root",
   "private": true,
   "devDependencies": {
+    "@angular-devkit/build-angular": "~15.1.0",
+    "@angular-devkit/schematics": "^15.1.1",
+    "@angular-eslint/eslint-plugin": "~15.0.0",
+    "@angular-eslint/eslint-plugin-template": "~15.0.0",
+    "@angular-eslint/template-parser": "~15.0.0",
+    "@angular/cli": "~15.1.0",
+    "@angular/compiler-cli": "~15.1.0",
+    "@angular/language-service": "~15.1.0",
+    "@nrwl/angular": "^15.5.1",
+    "@nrwl/eslint-plugin-nx": "15.5.1",
+    "@nrwl/jest": "15.5.1",
+    "@nrwl/linter": "15.5.1",
+    "@nrwl/storybook": "^15.5.1",
+    "@nxext/stencil": "^15.0.0",
+    "@stencil/angular-output-target": "^0.4.0",
+    "@stencil/core": "^2.17.1",
+    "@stencil/sass": "^2.0.2",
+    "@types/jest": "28.1.1",
+    "@types/node": "16.11.7",
+    "@typescript-eslint/eslint-plugin": "^5.36.1",
+    "@typescript-eslint/parser": "^5.36.1",
+    "eslint": "~8.15.0",
+    "eslint-config-prettier": "8.1.0",
+    "jest": "28.1.1",
+    "jest-environment-jsdom": "28.1.1",
+    "jest-preset-angular": "~12.2.3",
     "lerna": "^5.1.8",
-    "nx": "^14.4.2"
+    "ng-packagr": "^15.1.1",
+    "nx": "15.4.5",
+    "postcss-preset-env": "^7.8.3",
+    "ts-jest": "28.0.5",
+    "ts-node": "10.9.1",
+    "typescript": "^4.9.4"
+  },
+  "workspaces": [
+    "packages/*"
+  ],
+  "dependencies": {
+    "@angular/animations": "~15.1.0",
+    "@angular/common": "~15.1.0",
+    "@angular/compiler": "~15.1.0",
+    "@angular/core": "~15.1.0",
```

Comment:
Studio-Web is Angular 14.2. Will this mean we'll have to upgrade Studio-Web to 15.1 before we can combine it here?

Reply:
just based on the documentation, the jump from 14 to 15 seems pretty big...